### DISCUSSION TITLE
Capacitor with New Relic integration

### DISCUSSION THREAD

**Discussion by andysousa** [ionic-team/capacitor#4121] [Q&A]
Greetings,

Has anyone had any luck getting new relic integration with an Ionic/Angular app running in Capacitor?  It looks like a feature request at New Relic, but very little info on it.  Any direction would be greatly appreciated

Andy

**Comment by inspire22** [comment]
Just checking up on this, as I was hoping to get something similar working

**Reply by ndesai-newrelic** [reply]
it is available now.newrleic-capcitor-plugin https://github.com/newrelic/newrelic-capacitor-plugin

**Comment by arvin8** [comment]
Would be good to see this integration done...

**Reply by ndesai-newrelic** [reply]
it is available now . newrleic-capcitor-plugin https://github.com/newrelic/newrelic-capacitor-plugin

**Comment by ndesai-newrelic** [comment]
it is available now .newrleic-capcitor-plugin https://github.com/newrelic/newrelic-capacitor-plugin
